source/administration/config-settings.rst

```diff
@@ -1,6 +1,9 @@
 Configuration Settings
```

Comment:
This page doesn't render correctly on preview, but I'm not sure why.

Reply:
I used this render and it worked: http://rst.ninjs.org/, not sure what is happening here.